test/server-unit/xlsx.js

```diff
@@ -30,4 +30,26 @@ describe('util.js', () => {
         done(error);
       });
   });
+
+  it('Shoul check the number of rows to write in the xlsx file', () => {
+    const data = {
+      rows : [{ Firstname : 'Alice', Lastname : 'Bob' }],
+    };
+    const result = xlsx.find(data);
+    expect(result.length).to.be.equal(1);
+  });
+
+  it('Shoul check the number of rows to write in the xlsx file by specifying a key', () => {
+    const data = {
+      students : [
+        { name : 'Alice' },
+        { name : 'Bob' }],
+    };
+    // rowsDataKey is the specific key where the renderer will get data to write in the expected file
+    // it is used when the provided array doesn't have this key "rows"
+    const options = { rowsDataKey : 'students' };
+    const result = xlsx.find(data, options);
+    expect(result.length).to.be.equal(2);
```

Comment:
Good tests!

You can probably check directly the values, since `result` should be `students`, right?
```js
expect(result).to.deep.equal(data.students);
```